src/arrayMethodSort.js

```diff
@@ -4,9 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction =
+  compareFuncAsOriginalArraySortMethod) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(
    compareFunction = compareFuncAsOriginalArraySortMethod
  ) {
```

Reply:
Open and closing brackets should be on same column, inner content should have 2 spaces indentation from id (just like in html)